share/doc/wake/logging.md

```diff
@@ -0,0 +1,64 @@
+# Logging
+
+This article gives a high level overview of how wake logging works.
+Check out [print.wake](https://github.com/sifive/wake/blob/master/share/wake/lib/core/print.wake)
+for commented code.
+
+## How Logging works on Wake Execution commands
+
+Every `Job` will emit a stderr and a stdout.
+
+When writing your wake code, you can control where each goes with `setPlanStderr` and `setPlanStdout` before you `runJob`.
+We call this which `stream` each goes to (we often refer to a `stream` as a `logLevel`).
```

Comment:
I would suggest not formatting "stream" in monospace because it doesn't actually correspond to anything in code.